### PR TITLE
MOSIP-14191 : if refid already present in ABIS then process response.

### PR DESCRIPTION
**Pull request by MonobikashDas**

### CODE REVIEW

File: registration-processor/core-processor/registration-processor-abis-middleware-stage/src/main/java/io/mosip/registartion/processor/abis/middleware/stage/AbisMiddleWareStage.java [hunk 1]
```diff
@@ -1,697 +1,713 @@
...
+
+	}
+
+	private AbisResponseDto updateAbisResponseEntity(AbisCommonResponseDto abisCommonResponseDto, String response) {
+		AbisResponseDto abisResponseDto = new AbisResponseDto();
+
+		abisResponseDto.setId(RegistrationUtility.generateId());
+		abisResponseDto.setRespText(response.getBytes());
+
+		abisResponseDto.setStatusCode(isInsertSuccessResponse(abisCommonResponseDto) ?
```
Comment (gsasikumar) [inline registration-processor/core-processor/registration-processor-abis-middleware-stage/src/main/java/io/mosip/registartion/processor/abis/middleware/stage/AbisMiddleWareStage.java]: can u just rename this method as isInsertSuccess.

File: registration-processor/core-processor/registration-processor-abis-middleware-stage/src/main/java/io/mosip/registartion/processor/abis/middleware/stage/AbisMiddleWareStage.java [hunk 2]
```diff
@@ -1,697 +1,713 @@
...
+				AbisStatusCode.SUCCESS.toString() : AbisStatusCode.FAILED.toString());
+		abisResponseDto.setStatusComment(io.mosip.registartion.processor.abis.middleware.constants.FailureReason.getValueFromKey(abisCommonResponseDto.getFailureReason()));
+		abisResponseDto.setLangCode("eng");
```
Comment (gsasikumar) [inline registration-processor/core-processor/registration-processor-abis-middleware-stage/src/main/java/io/mosip/registartion/processor/abis/middleware/stage/AbisMiddleWareStage.java:630]: why do we have the language hard coded?

**Review by gsasikumar — CHANGES_REQUESTED** [review]

File: registration-processor/core-processor/registration-processor-abis-middleware-stage/src/main/java/io/mosip/registartion/processor/abis/middleware/stage/AbisMiddleWareStage.java (hunk 1 above)
Comment (MonobikashDas) [reply registration-processor/core-processor/registration-processor-abis-middleware-stage/src/main/java/io/mosip/registartion/processor/abis/middleware/stage/AbisMiddleWareStage.java]: Done

File: registration-processor/core-processor/registration-processor-abis-middleware-stage/src/main/java/io/mosip/registartion/processor/abis/middleware/stage/AbisMiddleWareStage.java (hunk 2 above)
Comment (MonobikashDas) [reply registration-processor/core-processor/registration-processor-abis-middleware-stage/src/main/java/io/mosip/registartion/processor/abis/middleware/stage/AbisMiddleWareStage.java:630]: langcode value is not being used anywhere from abis_response table,

**Review by gsasikumar — APPROVED** [review]

**Pull request merged**